crates/node/src/lib.rs

```diff
@@ -22,20 +22,40 @@ impl From<ChangedContent> for tailwindcss_oxide::ChangedContent {
 }
 
 #[derive(Debug, Clone)]
-#[napi(object)]
+#[napi]
```

Comment:
We are attaching a new function `scan_files`, so the simpler `#[napi(object)]` is not enough anymore. Using `#[napi(object)]` results in the missing function.